### DISCUSSION TITLE
1.4.2.3 build

### DISCUSSION THREAD

**Discussion by ghost** [Pryaxis/TShock#2316] [Q&A]
I followed the guide on Build how to make it but when i did the TshockAPI build wat should i do next?

**Comment by moisterrific** [comment]
If you've correctly built it, simply place `TShockAPI.dll` into the ServerPlugins folder.

**Reply by ghost** [reply]
If not hard can you help me because i had errors do you got discord?

**Comment by hakusaro** [comment]
You can just download unofficial builds now with this support following the instructions on the readme.

**Reply by ghost** [reply]
Can i have your discord so you can better guide me, i tried follow guide but it gave me error.
